web/public/locales/it.json

```diff
@@ -195,407 +235,1351 @@
+      "zoneName": "Camerun"
+    },
+    "CN": {
+      "zoneName": "Cina"
+    },
+    "CO": {
+      "zoneName": "Colombia"
+    },
+    "CR": {
+      "zoneName": "Costarica"
+    },
+    "CU": {
+      "zoneName": "Cuba"
+    },
+    "CV": {
+      "zoneName": "Capo Verde"
+    },
+    "CW": {
+      "zoneName": "Curaçao"
+    },
+    "CX": {
+      "zoneName": "Isola di Natale"
+    },
+    "CY": {
+      "zoneName": "Cipro"
+    },
+    "CZ": {
+      "zoneName": "Cechia"
+    },
+    "DE": {
+      "zoneName": "Germania"
+    },
+    "DJ": {
+      "zoneName": "Gibuti"
+    },
+    "DK": {
+      "zoneName": "Danimarca"
+    },
+    "DK-DK1": {
+      "countryName": "Danimarca",
+      "zoneName": "Danimarca occidentale"
+    },
+    "DK-DK2": {
+      "countryName": "Danimarca",
+      "zoneName": "Danimarca orientale"
+    },
+    "DK-BHM": {
+      "countryName": "Danimarca",
+      "zoneName": "Bornholm"
+    },
+    "DM": {
+      "zoneName": "Dominica"
+    },
+    "DO": {
+      "zoneName": "Repubblica Dominicana"
+    },
+    "DZ": {
+      "zoneName": "Algeria"
+    },
+    "EC": {
+      "zoneName": "Ecuador"
+    },
+    "EE": {
+      "zoneName": "Estonia"
+    },
+    "EG": {
+      "zoneName": "Egitto"
+    },
+    "EH": {
+      "zoneName": "Sahara occidentale"
+    },
+    "ER": {
+      "zoneName": "Eritrea"
+    },
+    "ES": {
+      "zoneName": "Spagna"
+    },
+    "ES-CE": {
+      "countryName": "Spagna",
+      "zoneName": "Ceuta"
+    },
+    "ES-IB-FO": {
+      "countryName": "Spagna",
+      "zoneName": "Formentera"
+    },
+    "ES-IB-IZ": {
+      "countryName": "Spagna",
+      "zoneName": "Ibiza"
+    },
+    "ES-IB-MA": {
+      "countryName": "Spagna",
+      "zoneName": "Maiorca"
+    },
+    "ES-IB-ME": {
+      "countryName": "Spagna",
+      "zoneName": "Minorca"
+    },
+    "ES-CN-FVLZ": {
+      "countryName": "Spagna",
+      "zoneName": "Fuerteventura/Lanzarote"
+    },
+    "ES-CN-GC": {
+      "countryName": "Spagna",
+      "zoneName": "Gran Canaria"
+    },
+    "ES-CN-HI": {
+      "countryName": "Spagna",
+      "zoneName": "El Hierro"
+    },
+    "ES-CN-IG": {
+      "countryName": "Spagna",
+      "zoneName": "La Gomera"
+    },
+    "ES-CN-LP": {
+      "countryName": "Spagna",
+      "zoneName": "La Palma"
+    },
+    "ES-CN-TE": {
+      "countryName": "Spagna",
+      "zoneName": "Tenerife"
+    },
+    "ES-ML": {
+      "countryName": "Spagna",
+      "zoneName": "Melilla"
+    },
+    "ET": {
+      "zoneName": "Etiopia"
+    },
+    "FI": {
+      "zoneName": "Finlandia"
+    },
+    "FJ": {
+      "zoneName": "Figi"
+    },
+    "FK": {
+      "zoneName": "Isole Falkland"
+    },
+    "FM": {
+      "zoneName": "Micronesia"
+    },
+    "FO": {
+      "zoneName": "Isole Faroe"
+    },
+    "FO-MI": {
+      "countryName": "Faroe Islands",
+      "zoneName": "Isole Principali"
+    },
+    "FO-SI": {
+      "countryName": "Faroe Islands",
+      "zoneName": "Suðuroy"
+    },
```

Comment:
Shouldn't the `countryName` here be Isole Faroe?

Reply:
Yes, you are right, I am going to commit it now